BibrecPage.py

```diff
@@ -27,6 +27,31 @@
 class BibrecPage (Page.Page):
     """ Implements the bibrec page. """
 
+    def split_summary(self, text, word_count=72):
+        """ Split summary into initial and remaining parts for toggling in the interface. """
+        if not text:
+            return None, None
+        words = text.split()
+        initial = ' '.join(words[:word_count])
+        remaining = ' '.join(words[word_count:]) if len(words) > word_count else None
+        return initial, remaining
+
+
+    def get_book_summary(self, book_id):
+        """ Get book summary directly from database. """
+        sql = "SELECT text FROM attributes WHERE fk_books = %s AND fk_attriblist = 520 LIMIT 1"
```

Comment:
there is no need to go to the database again here. the attributes have already been loaded and are available in dc.marcs

Reply:
yes that should work. Can you make the change?

Reply:
you could also use pop to remove it from the list. which would avoid having to exclude it later